### PR TITLE
Move APIs section into Database section

### PR DESCRIPTION
**Pull request by dannykng**
## Changes
- Move APIs section into Database section
- Rename "Auto-generated APIs" to "Data APIs"
- Rename "Supabase API"/"Admin API" to "Management API"

Screen Shot 2022-10-25 at 4 53 03 PM
Screen Shot 2022-10-25 at 4 53 24 PM
Screen Shot 2022-10-25 at 4 54 15 PM

### CODE REVIEW

File: apps/reference/docs/guides/api.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: api
-title: APIs
+title: Auto-generated APIs
```
Comment (kiwicopple) [inline apps/reference/docs/guides/api.mdx:3]: I discussed with Ant - perhaps we go with "Data APIs" (similar to Aurora) and then we use "Auto-generated" just in the tagline

For the Admin API, perhaps we go with "Management API"

Thoughts?

Comment (dannykng) [reply apps/reference/docs/guides/api.mdx]: Works for me!

File: apps/reference/docs/reference.mdx [hunk 2]
```diff
@@ -34,7 +34,7 @@ Reference documentation for the official Supabase client libraries, APIs, and to
     </div>
     {/* API */}
     <div class="col col--6">
-      <ButtonCard icon={useBaseUrl("/img/icons/api-icon.svg")} class="card" to={useBaseUrl("/reference/api")} style={{ height: '100%' }} title="Supabase API" description="Manage your Supabase organizations and projects."/>
+      <ButtonCard icon={useBaseUrl("/img/icons/api-icon.svg")} class="card" to={useBaseUrl("/reference/api")} style={{ height: '100%' }} title="Management API" description="Manage your Supabase projects programmatically."/>
```
Comment (dannykng) [inline apps/reference/docs/reference.mdx:37]: Had to make the description fit on one line...

File: apps/reference/docs/guides/api.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 id: api
-title: APIs
+title: Data APIs
 description: Auto-generating and Realtime APIs.
 sidebar_label: Overview
 ---
```
Comment (kiwicopple) [inline apps/reference/docs/guides/api.mdx:6]: In this file should we move the first sentence above the "Overview" heading and use the term "auto generating"?

Eg:

"Supabase auto-generates three types of API directly from your database schema."

## Overview

- REST ....

**Review by kiwicopple — APPROVED** [review]
one suggestion, otherwise this looks good @dannykng

File: apps/reference/docs/guides/api.mdx (hunk 3 above)
Comment (dannykng) [reply apps/reference/docs/guides/api.mdx:6]: Done—took out the "Overview" heading as it's unnecessary.

**Pull request merged**